Command_Library/src/main/java/dev/weaponboy/command_library/Subsystems/BlueRight.java

```diff
@@ -0,0 +1,106 @@
+package dev.weaponboy.command_library.Subsystems;
+
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+
+import dev.weaponboy.command_library.CommandLibrary.OpmodeEX.OpModeEX;
+import dev.weaponboy.nexus_pathing.Follower.follower;
+import dev.weaponboy.nexus_pathing.PathGeneration.commands.sectionBuilder;
+import dev.weaponboy.nexus_pathing.PathGeneration.pathsManager;
+import dev.weaponboy.nexus_pathing.PathingUtility.RobotPower;
+import dev.weaponboy.nexus_pathing.RobotUtilities.Vector2D;
+@Autonomous (name = "blueRight", group = "Autos")
+public class BlueRight extends OpModeEX {
+    double targetHeading;
+    pathsManager paths = new pathsManager();
+    follower follow = new follower();
+
+    private final sectionBuilder[] right1x1 = {
+            () -> paths.addPoints((new Vector2D(147, 360)), new Vector2D(151, 245))
```

Comment:
Like the names, bit confused about the positions though, the target point looks like its going to drop off the clip at the hang zone even if i reverse it. When you double check use this desmos, i made some updates to the position of the field cus it was a bit off: LINK: https://www.desmos.com/calculator/lvao3xiic0